content/guides/author-libraries.md

```diff
@@ -166,11 +222,29 @@ module.exports = {
 };
 ```
 
+W>At the moment of webpack 3.5.5, using the next configuration is not working properly as stated in the [issue 4824](https://github.com/webpack/webpack/issues/4824):
+
+```javascript
+module.exports = {
+    ...
+    output: {
+        ...
+        
+        libraryTarget: {
+            root:'_'
+        }
+    }
+    ...
+};
+```
+
+W> However, you can set libraryTarget.var='_' to expect the library as a global variable
+
 ### Add `libraryTarget`
 
-For widespread use of the library, we would like it to be compatible in different environments, i. e. CommonJS, AMD, Node.js and as a global variable.
+For widespread use of the library, we would like it to be compatible in different environments, i.e. CommonJS, AMD, Node.js and as a global variable.
 
-To make your library available for reuse, add `library` property in webpack configuration.
+To make your library available for reuse, add `library` property inside `output` in webpack configuration.
```

Comment:
g/`library` property/the `library` property
g/in webpack configuration/in the webpack configuration